api/apps/api/src/modules/projects/projects.controller.ts

```diff
@@ -148,6 +151,29 @@ export class ProjectsController {
     return await this.projectsService.remove(id);
   }
 
+  @ApiConsumesShapefile({ withGeoJsonResponse: false })
+  @ApiOperation({
+    description: 'Upload shapefile for project-specific protected areas',
```

Comment:
```suggestion
    description: 'Upload shapefile for project-specific planning unit grid',
```

Reply:
fixed in https://github.com/Vizzuality/marxan-cloud/pull/527/commits/adddd0c3f77118477dd1af5cc3759203775ed91e